js-structure_shapes/components/Pentagon/Pentagon.js

```diff
@@ -0,0 +1,9 @@
+import {getRandomColor} from "../../utils/randomColor.js"
+
+export default function Pentagon() {
```

Comment:
You make this a function, inspired me to make a reusable function out of it actually. I've learned something here 💯